packages/flutter_tools/lib/src/build_system/targets/android.dart

```diff
@@ -207,7 +207,7 @@ class AndroidAot extends AotElfBase {
 
   @override
   List<Source> get inputs => <Source>[
-    const Source.pattern('{FLUTTER_ROOT}/packages/flutter_tools/lib/src/build_system/targets/dart.dart'),
+    const Source.pattern('{FLUTTER_ROOT}/packages/flutter_tools/lib/src/build_system/targets/android.dart'),
```

Comment:
Can we rely on a build stamp so it invalidates on every upgrade?  Build system behavior could change without this file changing (could be a different PR).

Reply:
The engine version is used as a shortcut for this: https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/build_system/build_system.dart#L318

It should also include `flutter/version` though.

The goal of this input is just for local development, not to handle upgrades